Mathlib/Order/SuccPred/LinearLocallyFinite.lean

```diff
@@ -194,12 +194,13 @@ variable [SuccOrder ι] [IsSuccArchimedean ι] [PredOrder ι] {i0 i : ι}
 the range of `toZ`. -/
 def toZ (i0 i : ι) : ℤ :=
   dite (i0 ≤ i) (fun hi ↦ Nat.find (exists_succ_iterate_of_le hi)) fun hi ↦
-    -Nat.find (exists_pred_iterate_of_le (not_le.mp hi).le)
+    -Nat.find (@exists_pred_iterate_of_le _ _ P _ _ _ (not_le.mp hi).le)
```

Comment:
What's going on here?

Reply:
Couldn't work it out... :-(

Reply:
Ah, that looks better! Must be some change in elaboration